packages/common/src/content/_internal/ContentUiSchemaSettings.ts

```diff
@@ -45,13 +45,26 @@ export const buildUiSchema = async (
               { type: "weekly" },
               { type: "monthly" },
               { type: "yearly" },
-              // uncomment this when the manual option is available
-              // {
-              //   label: `option.manual.label`,
-              //   type: "manual",
-              //   helperActionIcon: "information-f",
-              //   helperActionText: "option.manual.helperActionText",
-              // },
+              {
+                type: "manual",
+                helperActionIcon: "information-f",
+                helperActionText: `{{${i18nScope}.fields.schedule.manual.helperActionText:translate}}`,
+              },
+            ],
+          },
+        },
+        // force update checkbox -- TODO: replace with button once available
+        {
+          type: "Control",
+          scope: "/properties/schedule/properties/_forceUpdate",
```

Comment:
Wait, we need to specify `/properties/_forceUpdate`? I would have figured that `/schedule/_forceUpdate` would have been enough.